pkg/querier/block.go

```diff
@@ -47,6 +48,9 @@ func NewBlockQuerier(cfg tsdb.Config, logLevel logging.Level, registerer prometh
 
 // Get implements the ChunkStore interface. It makes a block query and converts the response into chunks
 func (b *BlockQuerier) Get(ctx context.Context, userID string, from, through model.Time, matchers ...*labels.Matcher) ([]chunk.Chunk, error) {
+	log, ctx := spanlogger.New(ctx, "BlockQuerier.Get")
```

Comment:
Subsequent calls to the logger should be done like this:
`level.Error(log)`

Instead of:
`level.Error(util.Logger)`

I checked the rest of the changeset and this looks the only function also logging, so the LGTM should be just 1 line change away 😉

Reply:
Fixed!